src/Input/Input.js

```diff
@@ -67,6 +67,12 @@ export const styleSheet = createStyleSheet('MuiInput', (theme) => {
         appearance: 'none',
       },
     },
+    multiLine: {
+      resize: 'none',
+      'line-height': 'inherit',
```

Comment:
If we don't include line-height here the text gets all crunched together.

Are you saying here though that `line-height: 1` would be better than `line-height: inherit`? If so let's change this over. I'm really not very familiar with html/css, I generally just kind of "do what works".